accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -444,6 +444,10 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
               looking to find and fix errors quickly as they are building a
               component, while the Accessibility Assessment view provides
               explanation and suggested solutions for each issue reported.
+              To report an issue, or to learn about any existing checker issues, visit the{" "}
```

Comment:
I think this is confusing, because the previous line talks about reported issues, but we mean different kinds of reported issues here.  What about having a separate section for 'Feedback' rather than including this information in 'Usage'?  We'd also need to be clear this is talking about problems with the checker itself, or cases where the issues it reports don't seem to be accurate.
The link should probably be 'Equal Access git repository' and should not include the final period.